src/common/transformations/include/transformations/fp16_compression/mark_decompression_convert_constant_folding.hpp

```diff
@@ -58,3 +59,13 @@ class ov::pass::KeepConstantsPrecisionAndAddConverts : public MatcherPass {
     OPENVINO_RTTI("KeepConstantsPrecisionAndAddConverts", "0");
     KeepConstantsPrecisionAndAddConverts();
 };
+
+/**
+ * @ingroup ie_transformation_common_api
+ * @brief Prevents ConstantFolding for f16/bf16 Const + Convert to f32 to keep original FW float Constants
```

Comment:
I propose to explain the reason of having such transformation that we do not want to have large memory consumption for intermediate FP32 constants after const-folding and we want to avoid const-folding for them.
Also, please put the copy of this explanation into transformation pipeline of FEs in `frontend.cpp` above its call.

Reply:
done